clearpart-2.ks.in

```diff
@@ -0,0 +1,78 @@
+#version=DEVEL
+#test name: ignoredisk
+url @KSTEST_URL@
+install
+network --bootproto=dhcp
+
+bootloader --timeout=1
+clearpart --all --drives=sda --initlabel
+autopart
+
+keyboard us
+lang en_US.UTF-8
+timezone America/New_York --utc
+rootpw testcase
+shutdown
+
+%packages
+%end
+
+%pre
+# Doing all this here is a lot easier than trying to make virt tools do
+# what I want on the outside.  Also this'll run before storage.
+parted /dev/sda mklabel msdos
+parted --align=none /dev/sda mkpart primary 0 1G
+mkfs.ext4 /dev/sda1
+mkdir /sda
+mount /dev/sda1 /sda
+echo TEST > /sda/sentinel
+umount /sda
+rmdir /sda
+%end
+
+%pre
+# Doing all this here is a lot easier than trying to make virt tools do
+# what I want on the outside.  Also this'll run before storage.
+parted /dev/sdb mklabel msdos
+parted --align=none /dev/sdb mkpart primary 0 1G
+mkfs.ext4 /dev/sdb1
+mkdir /sdb
+mount /dev/sdb1 /sdb
+echo TEST > /sdb/sentinel
+umount /sdb
+rmdir /sdb
+%end
+
+%post --nochroot
+SYSROOT=${ANA_INSTALL_PATH:-/mnt/sysimage}
+
+mkdir /sda
+mount /dev/sda1 /sda
+if [[ $? == 0 ]]; then
```

Comment:
Is there a reason why you don't have a check that it wasn't mounted properly?

Reply:
/dev/sda1 was cleared, so I am not sure if it is an error if you cannot mount it anymore.